subsystem/elevator.py

```diff
@@ -0,0 +1,74 @@
+import wpilib
+from robotpy_toolkit_7407 import Subsystem
+from robotpy_toolkit_7407.motors.revmotor import SparkMax
+from robotpy_toolkit_7407.utils.units import meters
+from robotpy_toolkit_7407.sensors.limit_switches.limit_switch import LimitSwitch
+
+import constants
+import config
+#importing packages
+
+
+class Elevator(Subsystem): #elevator class
+    extendMotor: SparkMax = SparkMax(config.elevator_motor_extend_id)#motor that extends the arm
```

Comment:
```suggestion
    motor_extend: SparkMax = SparkMax(config.elevator_motor_extend_id)  # motor that extends the arm
```
Please use 'snake_case' (look it up) for naming variables.